Data/src/main/java/org/tribuo/data/csv/CSVLoader.java

```diff
@@ -220,7 +217,7 @@ public CSVLoader(OutputFactory<T> outputFactory) {
      * @return A datasource containing the csv data.
      * @throws IOException If the disk read failed.
      */
-    public ListDataSource<T> loadDataSource(Path csvPath, Set<String> responseNames) throws IOException {
+    public DataSource<T> loadDataSource(Path csvPath, Set<String> responseNames) throws IOException {
```

Comment:
You mention it in the commit message but not here, we should document that whatever order these responseNames are in will be used in the generated responses, so users should use an ordered set.

Reply:
Ok.